### ISSUE TITLE
Offline support

### ISSUE THREAD

**Issue by cheeaun** [cheeaun/busrouter-sg#12]
Make the web app work offline. At least show some text and load the stored localStorage data. Currently it's being blocked by the `gzip-enabled.js` dependency which seems kind of weird.

**Comment by cheeaun** [comment]
Partially works now since 3c24420bed1f4de9dffa16c9325d7e4918817707 . Still need to make **all** data to be stored offline.

**Comment by cheeaun** [comment]
Closing this for now.

Web app is offline-enabled. Making *all* data to be stored offline would use up too much storage and doesn't really make sense. Even if it can be done, the map tiles can't be stored offline anyway.

Will recreate a separate issue once everything is possible in near future.
